_events/2023-01-25-htcondor-osg.md

```diff
@@ -1,11 +1,11 @@
 ---
-short_title: Joint OSG All Hands Meeting and HTCondor Week July 10-14
-title: "Save the Dates for the Joint HTCondor Week - OSG All-Hands Meeting - 10-14 July 2023"
+short_title: Announcing Throughput Computing 2023 to be held July 10-14
+title: "Save the Dates for Throughput Computing 2023 - a joint HTCondor OSG All-Hands event"
 layout: events
 published: true
 
 excerpt: |
-    Joint OSG All Hands Meeting and HTCondor Week July 10-14 2023
+    Save the Dates for **Throughput Computing 2023** - a joint OSG All-Hands Meeting and HTCondor Week to be held July 10-14 2023 in Madison, Wisconsin
```

Comment:
Why is Dates capitalized? Why is the order of HTCondor Week and OSG All-Hands Meeting reversed here (didn’t Todd want top billing?)?

I think it is traditional to put a comma after the dates: “July 10–14, 2023”, or go with a more international style with “10–14 July 2023”.

Reply:
I added - and also lower cased Dates and added a comma before 2023